XML/TestCases/FunctionalTests-DPDK.xml

```diff
@@ -173,4 +173,22 @@
         <Tags>dpdk,dpdk-core,network,hv_netvsc,sriov</Tags>
         <Priority>2</Priority>
     </test>
+    <test>
+        <testName>DPDK-PMDS-BASIC-CHECK</testName>
+        <testScript>DPDK-PMDS-BASIC-CHECK.ps1</testScript>
+        <setupType>OneVM3NIC</setupType>
+        <AdditionalHWConfig>
+            <Networking>SRIOV</Networking>
+        </AdditionalHWConfig>
+        <files>.\Testscripts\Linux\utils.sh,.\Testscripts\Linux\dpdkSetup.sh,.\Testscripts\Linux\dpdkUtils.sh,.\Testscripts\Linux\dpdkPmdsBasicCheck.sh,.\Testscripts\Linux\failsafe.sh</files>
+        <TestParameters>
+            <param>dpdkSrcLink="DPDK_SOURCE_URL"</param>
+            <param>pmds=mlx,vdev_netvsc,tap,failsafe,netvsc</param>
+        </TestParameters>
+        <Platform>Azure</Platform>
+        <Category>Functional</Category>
+        <Area>DPDK</Area>
+        <Tags>dpdk,dpdk-core,network,hv_netvsc,sriov</Tags>
+        <Priority>2</Priority>
```

Comment:
@abhimarathe Please confirm if this priority 2 is ok for this test.

Reply:
Yes priority 2 is fine. Right now we have failsafe as priority 0.